scripts/dotnet-cli-build/MsiTargets.cs

```diff
@@ -52,14 +55,24 @@ private static void AcquireWix(BuildTargetContext c)
             Directory.CreateDirectory(WixRoot);
 
             c.Info("Downloading WixTools..");
-            // Download Wix version 3.10.2 - https://wix.codeplex.com/releases/view/619491
-            Cmd("powershell", "-NoProfile", "-NoLogo",
-                $"Invoke-WebRequest -Uri https://wix.codeplex.com/downloads/get/1540241 -Method Get -OutFile {WixRoot}\\WixTools.zip")
-                    .Execute()
-                    .EnsureSuccessful();
+
+            DownloadFile($"https://dotnetcli.blob.core.windows.net/build/wix/wix.{WixVersion}.zip", Path.Combine(WixRoot, "WixTools.zip"));
```

Comment:
We're building a URL here, not a path, so I don't think Path.Combine is the best thing to do here.